### PR TITLE
track completion_kwargs in Completion method

### PR DESCRIPTION
**Pull request by jerpint**

### CODE REVIEW

File: buster/completers/base.py [hunk 1]
```diff
@@ -171,6 +173,7 @@ def encode_df(df: pd.DataFrame) -> dict:
             "matched_documents": self.matched_documents,
             "answer_relevant": self.answer_relevant,
             "question_relevant": self.question_relevant,
+            "completion_kwargs": self.completion_kwargs,
```
Comment (hbertrand) [inline buster/completers/base.py:176]: The risk I can see is if `completion_kwargs` contains non-jsonable fields.

Comment (jerpint) [reply buster/completers/base.py:176]: yes, for now it shouldnt with openai kwargs but in the future it's something to be mindful of indeed, i dont think we should worry about it right now though

Comment (jerpint) [reply buster/completers/base.py:176]: Great, looks like we are ready to merge

Comment (jerpint) [reply buster/completers/base.py:176]: @hbertrand can you see this message?

Comment (jerpint) [reply buster/completers/base.py:176]: New comment from me!

**Pull request merged**